src/stories/Foundation/Icon/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+  /* Your styles here */
+`;
+
+export interface IconProps {
+  className?: string;
+  icon: any;
```

Comment:
I assume this is the component you are working on and that these will be fixed? 😊

Reply:
Yes yes, not finished ofc!